ansible/configs/sap-hana-manual/requirements.yml

```diff
@@ -0,0 +1,2 @@
+---
```

Comment:
If this isn't used, can you just delete the file?

Reply:
Done